kaniko/README.md

```diff
@@ -60,7 +62,11 @@ spec:
     arguments:
     - name: IMAGE
       value: us.gcr.io/my-project/my-app
+    - name: DOCKERFILE
+      value: ./Dockerfile
```

Comment:
In order to be consistent, you'd better set the value /workspace/myfolder/Dockerfile to DOCKERFILE. BTW, the default value should be /workspace/Dockerfile. I doubt ./Dockerfile will point to no file.

Reply:
ok, i'll  use the full path there subject to resolving the comment below..  

 i recall the `./Dockerfile`  did work for me fine

Reply:
If we set CONTEXT, then we can either use a absolute path or relative path for DOCKERFILE, so ./Dockerfile is fine. If we so not set CONTEXT, I think we have to set DOCKERFILE to /workspace/Dockerfile. Let me verify one more time.